index.js

```diff
@@ -376,4 +407,41 @@ export default class OneSignal {
 
         RNOneSignal.removeExternalUserId();
     }
+
+    static addTriggers(triggers) {
+        if (!checkIfInitialized()) return;
+
+        RNOneSignal.addTriggers(triggers);
+    }
+
+    static addTrigger(key, object){
+        if (!checkIfInitialized()) return;
+
+        RNOneSignal.addTrigger(key, object);
+    }
+
+    static removeTriggersForKeys(keys) {
+        if (!checkIfInitialized()) return;
+
+        RNOneSignal.removeTriggersForKeys(keys);
+    }
+
+    static removeTriggerForKey(key) {
+        if (!checkIfInitialized()) return;
+
+        RNOneSignal.removeTriggerForKey(key);
+    }
+
+    static getTriggerValueForKey(key) {
+        if (!checkIfInitialized()) return;
+
+        RNOneSignal.getTriggerValueForKey(key);
```

Comment:
This needs to be a callback like `getTags` or return a promise like `userProvidedPrivacyConsent`.

Since `getTriggerValueForKey` is closer to `userProvidedPrivacyConsent` I think it should be a promise, since it doesn't have a callback and just returns a value at the native level.

Reply:
fixed @jkasten2